core/renderers/geras/path_object.js

```diff
@@ -34,11 +36,19 @@ goog.require('Blockly.utils.object');
  * An object that handles creating and setting each of the SVG elements
  * used by the renderer.
  * @param {!SVGElement} root The root SVG element.
+ * @param {!Blockly.blockRendering.ConstantProvider} constants The renderer's
```

Comment:
This should be ``Blockly.geras.ConstantProvider`` no?

Reply:
No, because it's a `blockRendering.ConstantProvider` on the interface.  But I will eventually make an interface for the constants provider as well, and then can change this.